corgi/core/fixups.py

```diff
@@ -261,73 +86,22 @@
-        "cpe:/a:redhat:satellite_utils:6.11::el8",
-        "cpe:/a:redhat:satellite_capsule:6.11::el7",
-        "cpe:/a:redhat:satellite:6.11::el8",
-        "cpe:/a:redhat:rhel_satellite_client:6::el9",
-        "cpe:/a:redhat:satellite_maintenance:6.11::el8",
-        "cpe:/a:redhat:satellite_capsule:6.11::el8",
-        "cpe:/a:redhat:rhel_satellite_client:6::el6",
-        "cpe:/a:redhat:rhel_satellite_client:6::el7",
-    ],
-    "rhn_satellite_6.12": [
-        "cpe:/a:redhat:satellite_utils:6.12::el8",
-        "cpe:/a:redhat:satellite_maintenance:6.12::el8",
-        "cpe:/a:redhat:satellite:6.12::el8",
-        "cpe:/a:redhat:satellite_capsule:6.12::el8",
-    ],
-    "rhn_satellite_6.13": [
-        "cpe:/a:redhat:satellite:6.13::el8",
-        "cpe:/a:redhat:satellite_capsule:6.13::el8",
-        "cpe:/a:redhat:satellite_maintenance:6.13::el8",
-        "cpe:/a:redhat:satellite_utils:6.13::el8",
-    ],
-    "rhods-1.20": ["cpe:/a:redhat:openshift_data_science:1.20::el8"],
-    "rhods-1.21": ["cpe:/a:redhat:openshift_data_science:1.21::el8"],
-    "rhods-1.23": ["cpe:/a:redhat:openshift_data_science:1.23::el8"],
-    "rhods-1.24": ["cpe:/a:redhat:openshift_data_science:1.24::el8"],
+    # None of the matching variants are live
     "rhosc-1": ["cpe:/a:redhat:openshift_sandboxed_containers:1.3.0::el8"],
-    "rhos_devspaces-3": ["cpe:/a:redhat:openshift_devspaces:3::el8"],
-    "rhosdt-2": ["cpe:/a:redhat:openshift_distributed_tracing:2.3::el8"],
+    # Not getting match with pattern matching, excluded from brew_tag matching because it shadows
+    # dts-11.0.z
     "rhscl-3.8.z": [
         "cpe:/a:redhat:rhel_software_collections:3::el6",
         "cpe:/a:redhat:rhel_software_collections:3::el7",
     ],
-    "rhui-4": ["cpe:/a:redhat:rhui:4::el8"],
-    "rodoo-1": ["cpe:/a:redhat:run_once_duration_override_operator:1.0::el8"],
-    "rosa-cli-1": ["cpe:/a:redhat:openshift_service_on_aws:1::el8"],
-    "snr-0.4.z": ["cpe:/a:redhat:workload_availability_self_node_remediation:0.4::el8"],
-    "srvcom-1": ["cpe:/a:redhat:serverless:1.0::el8", "cpe:/a:redhat:serverless:1.25::el8"],
-    "stf-1.4": ["cpe:/a:redhat:service_telemetry_framework:1.4::el8"],
-    "stf-1.5": ["cpe:/a:redhat:service_telemetry_framework:1.5::el8"],
-    "wto-1.5": ["cpe:/a:redhat:webterminal:1.5::el8"],
-    "rhcertification-6": ["cpe:/a:redhat:certifications:1::el6"],
-    "rhcertification-7": ["cpe:/a:redhat:certifications:1::el7"],
-    "rhcertification-8": ["cpe:/a:redhat:certifications:1::el8"],
-    "rhcertification-9": ["cpe:/a:redhat:certifications:1::el9"],
 }
 
 
-def cpe_lookup(product_stream_name: str) -> list[str]:
-    """Temporary cpe fixup"""
-    return supported_stream_cpes.get(product_stream_name, [])
+def cpe_lookup(product_stream_name: str) -> set[str]:
+    """Manual CPE overrides for streams which cannot be matched with variants"""
+    return set(supported_stream_cpes.get(product_stream_name, []))
```

Comment:
FYI you can also use set literal syntax in the dict above, instead of having to convert the values to a set at the end. Any way you prefer is fine:

```suggestion
def cpe_lookup(product_stream_name: str) -> set[str]:
    """Manual CPE overrides for streams which cannot be matched with variants"""
    return supported_stream_cpes.get(product_stream_name, set())
```

And then above would be something like:
```
supported_stream_cpes = {
    # curly brace above creates a dict, because it has colons / keys and values
    # curly brace below creates a set, because it has no colons / keys, only values
    "stream_A": {"value_1", "value_2"},
    # Can also use set() constructor syntax
    "stream_B": set("value_3", "value_4"),
}
```